internal/schemautil/schemautil.go

```diff
@@ -161,7 +161,15 @@ func EmptyObjectDiffSuppressFuncSkipArrays(s map[string]*schema.Schema) schema.S
 	return func(k, old, new string, d *schema.ResourceData) bool {
 		for _, key := range skipKeys {
 			if strings.Contains(k, fmt.Sprintf(".%s.", key)) {
-				return false
+				if strings.HasSuffix(k, ".ip_filter.#") {
+					if list, ok := d.Get(strings.TrimSuffix(k, ".#")).([]interface{}); ok {
```

Comment:
I would like to add some comments here, code of this function is pretty complex, and with this change, it is getting even more so.